receiver/technitium_api.py

```
@@ -0,0 +1,650 @@
+    def set_enricher(self, enricher):
+        self._enricher = enricher
+
+    def reload_config(self):
+        """Re-read settings from DB/env. Restart polling if changed."""
+        old_enabled = self.enabled
+        old_hosts = {s['id']: s['host'] for s in self.servers}
+
+        for sess in self._sessions.values():
+            try:
+                sess.close()
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)